packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -249,12 +248,13 @@ class _ToolbarRenderBox extends RenderShiftedBox {
 
 /// Draws a single text selection handle with a bar and a ball.
 class _TextSelectionHandlePainter extends CustomPainter {
-  const _TextSelectionHandlePainter();
+  const _TextSelectionHandlePainter(this.color);
+  final Color color;
```

Comment:
This parameter in material's _TextSelectionHandlePainter is a named parameter, so probably make this one named too for consistency.

Also, maybe they should both be required and non-null?  I'm not sure why the material one isn't like that already, but maybe there's a reason I'm not seeing.